Makefile-rpm.mk

```diff
@@ -0,0 +1,29 @@
+NAME    := raft
+SRC_EXT := gz
+SOURCE   = v$(VERSION).tar.$(SRC_EXT)
```

Comment:
Since we are building the archive from git when not using "official" releases, let's normalize the name here to what the name would be in a release: `raft-$(VERSION).tar.$(SRC_EXT)`.

Reply:
Will be in next update